### ISSUE TITLE
There are several mistakes in writing in your paper? (论文中的几处笔误)

### ISSUE THREAD

**Issue by Luoluoluoplus7** [lehaifeng/DASNet#15]
Hi, 
I've read your paper and I think that the number "0" was missed in the maximum functions of Eq.(5) and Eq.(6). Am I right? (论文中公式5和6中求max的地方忘写零了？)

And I'm also a little confused about your WDMC loss function. 
I can't find any weights in "class DContrastiveLoss" in your code. Is this "DContrastiveLoss" the implementation of your WDMC loss?

Thank you~

**Comment by Luoluoluoplus7** [comment]
By the way, how many epochs are needed for training BCDDataset?
The convergence speed of BCDD is so slow....

**Comment by dsagsdfgfdh** [comment]
We trained 40 epochs for each data set

**Comment by zyp19** [comment]
我也觉得应该是缺少了零
